src/main/resources/view/ServiceTypeAndRoomTypePanel.fxml

```diff
@@ -8,33 +8,31 @@
 
 <?import javafx.geometry.Insets?>
 <?import javafx.scene.layout.StackPane?>
-<VBox xmlns:fx="http://javafx.com/fxml/1" xmlns="http://javafx.com/javafx/8">
-    <TabPane tabMinWidth="183" prefWidth="500.0" tabClosingPolicy="UNAVAILABLE" VBox.vgrow="ALWAYS">
-        <tabs>
-            <Tab text="Service Type">
-                <content>
-                    <VBox minWidth="340.0" prefWidth="340.0" VBox.vgrow="ALWAYS">
-                        <padding>
-                            <Insets top="10" right="10" bottom="10" left="10"/>
-                        </padding>
-                        <children>
-                            <StackPane fx:id="serviceTypeListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
-                        </children>
-                    </VBox>
-                </content>
-            </Tab>
-            <Tab text="Room Type">
-                <content>
-                    <VBox minWidth="340.0" prefWidth="340.0" VBox.vgrow="ALWAYS">
-                        <padding>
-                            <Insets top="10" right="10" bottom="10" left="10"/>
-                        </padding>
-                        <children>
-                            <StackPane fx:id="roomTypeListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
-                        </children>
-                    </VBox>
-                </content>
-            </Tab>
-        </tabs>
-    </TabPane>
-</VBox>
+<TabPane fx:id="tabPane" xmlns:fx="http://javafx.com/fxml/1" xmlns="http://javafx.com/javafx/8" tabMinWidth="183" prefWidth="500.0" tabClosingPolicy="UNAVAILABLE" VBox.vgrow="ALWAYS">
+    <tabs>
+        <Tab text="Service Type">
+            <content>
+                <VBox minWidth="340.0" prefWidth="340.0" VBox.vgrow="ALWAYS">
+                    <padding>
+                        <Insets top="10" right="10" bottom="10" left="10"/>
+                    </padding>
+                    <children>
```

Comment:
I have changed the width of the panels. Its already been updated. Can go ahead with merging. Just make sure you keep my changes too.

Reply:
And also does this automatically switch tabs? Whenever we update booking or reservation, does it automatically change the tab?